articles/machine-learning/data-science-virtual-machine/dsvm-samples-and-walkthroughs.md

```diff
@@ -23,7 +23,7 @@ Azure Data Science Virtual Machines (DSVMs) include a comprehensive set of sampl
 
 ## Prerequisites
 
-In order to run these samples, you must have provisioned a Data Science Virtual Machine. See the quickstarts for [Windows](./provision-vm.md) and [Ubuntu](./dsvm-ubuntu-intro.md).
+In order to run these samples, you must have provisioned a Data Science Virtual Machine. See the quickstarts for Windows 2016 and [Ubuntu](./dsvm-ubuntu-intro.md).
```

Comment:
I'd suggest changing this to: 

"In order to run these samples, you must have [provisioned an Ubuntu Data Science Virtual Machine](./dsvm-ubuntu-intro.md)"